src/patterns/dosdp-patterns/entity_attribute_partOf.yaml

```diff
@@ -0,0 +1,68 @@
+pattern_name: entity_attribute
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/entity_attribute.yaml
+
+description: '
+
+  Basic pattern for traits that captures a quality, such as size or rate which is a
+  characteristic of an entity, such as an anatomical entity or a biological process.'
+
+classes:
+  quality: PATO:0000001
+  thing: owl:Thing
+
+relations:
+  part_of: BFO:0000050
+  characteristic_of: RO:0000052
+  characteristic_of_part_of: RO:0002314
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+  xref: oio:hasDbXref
+
+vars:
+  attribute: "'quality'"
+  entity: "'thing'"
+
```

Comment:
Add data_list_vars to make this pattern consistent with the `entity_attribute` template.
> data_list_vars:
>   exact_synonyms: xsd:string
>   xrefs: xsd:string